### PR TITLE
doc/tutorial: tutorial improvements

### PR DESCRIPTION
**Pull request by minaelee**
- fix typo of `micro01`..`04` to `micro1`..`4` in several places
- fix typo of `microcloud init` to `microcloud join` in output for joining members
- use `cluster member` when appropriate
- simplify & clarify text
- reduce parentheses and future tense
- note that VMs should not be used in production
- remove mention of needing minimum 3 VMs
- update screengrabs of the init and join processes

### CODE REVIEW

File: doc/tutorial/get_started.md [hunk 1]
```diff
@@ -267,48 +264,50 @@ Complete the following steps:
...
+       lxc exec micro2 microcloud join
 
-         ```{tip}
+   ```{tip}
+   Open three additional terminals to run the commands concurrently.
+   ```
 
-         Open up three additional terminals to run the commands concurrently.
-         ```
+   In each joining cluster member, select the address for MicroCloud's internal traffic that was chosen on `micro1`.
```
Comment (roosterfish) [inline doc/tutorial/get_started.md:276]: "from the same subnet that was chosen on `micro01`"?

Comment (roosterfish) [reply doc/tutorial/get_started.md:276]: Maybe I am wrong but the sentence sounds like you should select the same address for MicroCloud's internal traffic that was already chosen on `micro1`.
But in fact it's a different address in the same subnet.

Comment (minaelee) [reply doc/tutorial/get_started.md]: You are totally right. I updated it to use the same wording as on the initiating cluster member:
>In each joining cluster member, select the listed IPv4 address for MicroCloud's internal traffic.

I also added to the commit an updated screengrab of the join process that shows the address options, to help make it more clear. Thanks!

Comment (roosterfish) [reply doc/tutorial/get_started.md]: Thx!

**Review by roosterfish — APPROVED** [review]

**Pull request merged**